lib/ActionFactory.php

```diff
@@ -27,8 +27,6 @@ public function create($name) {
 			throw new \Exception('Action ' . $name . ' does not exist!');
 		}
 
-		require_once $filePath;
-
```

Comment:
Tbh, I don't understand why should we not use require_once in this case.

Reply:
This commit move the include calls to `lib/` and later an autoloader is introduced so it's not necessary to manually include files anymore.